vue.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    "airbnb-typescript",
+    "eslint-config-prettier",
+    "plugin:vue/vue3-strongly-recommended",
+    "./base.js",
+  ],
+  env: {
+    browser: true,
+  },
+  parser: "vue-eslint-parser",
+  parserOptions: {
+    ecmaFeatures: {
+      globalReturn: false,
+      impliedStrict: false,
+      jsx: false
+    },
+    parser: "@typescript-eslint/parser"
```

Comment:
I think you are declaring 2 diff parsers here, no? 
`parser: "vue-eslint-parser"`
`parser: "@typescript-eslint/parser"`

Reply:
I think we can remove the `@typescript-eslint/parser`